src/key.rs

```diff
@@ -460,32 +469,21 @@ pub fn generate(
 
     let txn = yubikey.begin_transaction()?;
 
-    templ[3] = slot.into();
+    let templ = [0, Ins::GenerateAsymmetric.code(), 0, slot.into()];
```

Comment:
Sidebar but: wonder if we could come up with a better representation for these templates